dev/ProgressBar/TestUI/ProgressBarPage.xaml

```diff
@@ -27,12 +27,12 @@
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
 
-        <StackPanel Grid.ColumnSpan="3" Orientation="Horizontal">
+        <StackPanel Grid.ColumnSpan="3" Orientation="Horizontal" Spacing="8">
```

Comment:
>Spacing [](http://example.com/codeflow?start=65&length=7)

Spacing is not available on RS2 which explains why all your tests are failing down level